kubernetes-client/src/main/java/io/fabric8/kubernetes/client/BatchAPIGroupClient.java

```diff
@@ -15,12 +15,12 @@
  */
 package io.fabric8.kubernetes.client;
 
-import io.fabric8.kubernetes.api.model.batch.DoneableJob;
-import io.fabric8.kubernetes.api.model.batch.Job;
-import io.fabric8.kubernetes.api.model.batch.JobList;
+import io.fabric8.kubernetes.api.model.batch.*;
```

Comment:
We should ignore this practice, while doing the 3.9 upgrade, I tried to remove it as much as possible, this creates lots of problems when doing the upgrade, let's ignore * and import only which is required.

Reply:
I think it's done automatically while importing things in IntelliJ. Not sure if it's a bad practice or not.